admin/members.php

```diff
@@ -26,6 +26,7 @@
 */
 
 $_language->readModule('members');
+$_language->readModule('rank_special', true);
```

Comment:
The language file is also added to other admin/.php files.
Merging would create duplicate entries.